cmd/minikube/cmd/delete.go

```diff
@@ -69,7 +70,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	if err = cluster.DeleteHost(api); err != nil {
 		switch err := errors.Cause(err).(type) {
 		case mcnerror.ErrHostDoesNotExist:
-			out.T(out.Meh, `"{{.name}}" cluster does not exist`, out.V{"name": profile})
+			out.T(out.Meh, `"{{.name}}" cluster does not exist. Proceeding ahead with cleanup.`, out.V{"name": err.Name})
```

Comment:
Just curious: why the change from profile to err.Name?

Reply:
Just that this seems to align more with the error being returned i.e. `ErrHostDoesNotExist`. It could be that the profile exists but the host itself has been deleted. Would be happy to revert it back though.